src/pages/basics/introduction.js

```diff
@@ -11,42 +11,40 @@ const IntroductionPage = () => (
     <h3>What is a shader?</h3>
     <p>
       A shader is a program that is primarily used to shade an image, generating
-      color information while applying any lighting and darkening effects on
-      that image.
+      color information while applying any lighting effects on
+      said image.
     </p>
     <p>
-      While shaders are still used for this purpose, they can now also be used
-      to perform other types of graphical and compute tasks.
+      While shaders are mainly used for this purpose, they can now also be used
+      to perform other types of computational tasks.
```

Comment:
compute tasks instead of computational?

Reply:
As mentioned, do take care when trying to add detail and remove detail. Sometimes, details don't matter, and here, it certainly doesn't.